docs/user_guide/index.rst

```diff
@@ -0,0 +1,94 @@
+.. _intro_user_guide:
+
+User guide
+==========
+
+The user guide is organised through the following sections:
+
+.. grid:: 3
+    :gutter: 1
+
+    .. grid-item-card::
+        :text-align: center
+        :link: 1_getting_started_hydromt/index
+        :link-type: doc
+
+        :octicon:`rocket;10em`
+        +++
+        Getting started with HydroMT
+
+    .. grid-item-card::
+        :text-align: center
+        :link: 2_sbm_model/index
+        :link-type: doc
+
+        :octicon:`file-directory-open-fill;10em`
+        +++
+        Wflow SBM
+
+    .. grid-item-card::
+        :text-align: center
+        :link: 3_sediment_model/index
+        :link-type: doc
+
+        :octicon:`file-directory-open-fill;10em`
+        +++
+        Wflow Sediment
+
+    .. grid-item-card::
+        :text-align: center
+        :link: 4_pre_and_post_processing/index
+        :link-type: doc
+
+        :octicon:`graph;10em`
+        +++
+        Processing and Visualization
+
+    .. grid-item-card::
+        :text-align: center
+        :link: 5_setup_methods/index
+        :link-type: doc
+
+        :octicon:`cpu;10em`
+        +++
+        Technical Description
+
+    .. grid-item-card::
+        :text-align: center
+        :link: 6_migration_guide/index
+        :link-type: doc
+
+        :octicon:`arrow-switch;10em`
+        +++
+        Migration Guide
+
+
+With the **HydroMT-Wflow plugin**, users can easily benefit from the rich set of tools of the
```

Comment:
I almost missed the text. Maybe with smaller icons this will be less of an issue, but I'm hesitating to put the text first and then the icon table. What do you think?